x/distribution/keeper/hooks.go

```diff
@@ -187,3 +188,13 @@ func (h Hooks) BeforeDelegationRemoved(_ context.Context, _ sdk.AccAddress, _ sd
 func (h Hooks) AfterUnbondingInitiated(_ context.Context, _ uint64) error {
 	return nil
 }
+
+func (h Hooks) AfterConsensusPubKeyUpdate(ctx sdk.Context, _, _ cryptotypes.PubKey, rotationFee sdk.Coin) error {
+	feePool, err := h.k.FeePool.Get(ctx)
+	if err != nil {
+		return err
+	}
+
+	feePool.CommunityPool = feePool.CommunityPool.Add(sdk.NewDecCoinsFromCoins(rotationFee)...)
```

Comment:
Was the money sent before? 

I think it'd be beneficial to have send + state update in the same place rather than in two separate locations.

Reply:
I think he's doing this to avoid leaking distribution into x/staking?

```
err = k.Keeper.bankKeeper.SendCoinsFromAccountToModule(ctx, sdk.AccAddress(valAddr), types.DistributionModuleName, sdk.NewCoins(params.KeyRotationFee))
```

I agree that this doesn't look good. We should look into adapting this to x/protocolpool instead of x/distribution right?

Reply:
+1 we should deprecate (deprecated comment) everything  community pool related in distribution (cc @likhita-809) to make it more clear as well.

Reply:
++ to @julienrbrt's comment

Reply:
Thanks for the feedback all, makes sense to me!

Reply:
Since the protocolpool is separately added as a module, these changes are not needed any more, updated the msg server to send the tokesn direct to the protocolpool module.